x/staking/keeper/keeper.go

```diff
@@ -93,6 +93,16 @@ type Keeper struct {
 	LastValidatorPower collections.Map[[]byte, gogotypes.Int64Value]
 	// Params key: ParamsKeyPrefix | value: Params
 	Params collections.Item[types.Params]
+	// ValidatorConsPubKeyRotationHistory: consPubkey rotation history by validator
+	ValidatorConsPubKeyRotationHistory collections.Map[collections.Pair[[]byte, uint64], types.ConsPubKeyRotationHistory]
+	// BlockConsPubKeyRotationHistory: consPubkey rotation history by height
+	BlockConsPubKeyRotationHistory collections.Map[uint64, types.ConsPubKeyRotationHistory]
```

Comment:
Would it be possible to combine these two into a `collections.IndexedMap`? Using a `Multi` index:
```
// Multi defines the most common index. It can be used to create a reference between
// a field of value and its primary key. Multiple primary keys can be mapped to the same
// reference key as the index does not enforce uniqueness constraints.
```

Reply:
Agree